storage/timecache/export_test.go

```diff
@@ -15,12 +15,21 @@ func (tc *TimeCache) KeyTime(key string) (time.Time, bool) {
 
 	val, ok := tc.data[key]
 
-	return val, ok
+	return val.timestamp, ok
+}
+
+func (tc *TimeCache) KeySpan(key string) (time.Duration, bool) {
+	tc.mut.Lock()
+	defer tc.mut.Unlock()
+
+	val, ok := tc.data[key]
+
+	return val.span, ok
```

Comment:
actually, it can not, since the tc.data is of type `map[string]span` (map of values not pointers) which defaults the returned object `val` to an empty span